test/basic_get_save_v1_test.py

```diff
@@ -347,3 +339,45 @@ def test_handles(self):
         self.assertTrue('genbank_handle_ref' in genome)
         hid2 = genome['genbank_handle_ref']
         self.assertNotEqual(hid1, hid2)
+
+    @log
+    def test_save_genome_with_close_genome(self):
```

Comment:
Hi Uma - looking at this test, is there a way to force a non-float closeness value to make sure that the error checking works?